### ISSUE TITLE
Update healthcheck.sh as rabbitmqctl node_health_check is deprecated

### ISSUE THREAD

**Issue by hypr2771** [bitnami/containers#4230]
### Name and Version

bitnami/rabbitmq:10.1.14

### What steps will reproduce the bug?

Run a pod using this chart, healthcheck warns should start showing up each seconds.

### What is the expected behavior?

We should use the correct way of doing healthchecks (depending on whether we are in cluster or standalone mode), as stated in [the official RabbitMQ documentation](https://www.rabbitmq.com/monitoring.html#health-checks).

### What do you see instead?

Following messages are logged each seconds:

```
[warning] <0.1811.50> rabbitmqctl node_health_check and its HTTP API counterpart are DEPRECATED. See https://www.rabbitmq.com/monitoring.html#health-checks for replacement options.
```

### Additional information

This is a port of https://github.com/bitnami/bitnami-docker-rabbitmq/issues/122 from before the merge into this single repository.

**Comment by marcosbc** [comment]
Hi @hypr2771, sorry for the delay in getting back to you. Just wanted to let you know that we have created a new internal task for looking into this (the other one got closed after a cleanup). Unfortunately I cannot provide an ETA for when it will be fixed.

**Comment by fevisera** [comment]
Hi @hypr2771, we have released this new version [bitnami/rabbitmq:11.2.2](https://github.com/bitnami/charts/pull/13991) which should fix that healthcheck warning.

Please, check it and let us know if that warning is gone, so we can close this issue.

Thanks.
